File: BE/Resources/ProductResource.cs
using supemarket.models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Supemarket.Resources
{
    public class ProductResource
    {
        public int Id { get; set; }
        public String name { get; set; } = "NULL";
        public double price { get; set; } = 0;
        public string expiry_date { get; set; }
        public string production_date { get; set; }
        public int number_of_items { get; set; } = 12;
        public ProductTypesModel category { get; set; } = ProductTypesModel.chips;
        public String made_in { get; set; }
        public double width { get; set; }
        public double height { get; set; }
        public double length { get; set; }
        public Boolean active { get; set; }
    }
}

File: FE/SuperMarketAngular/src/app/models/Resources/ProductResource.ts

export class ResourceProduct {

   Id : number = 0 ;
   name :string = "NULL";
   price: number = 0;
   expiry_date: Date = new Date() ;
   production_date: Date = new Date() ;
   number_of_items :number = 12;
   category:number = 0;
   made_in:string = "" ;
   width:number = 0 ;
   height:number = 0 ;
   length:number = 0 ;
   active:Boolean = false ;

   ResourceProduct(){
       
   }

}